Code/src/main/java/nl/utwente/viskell/ui/CircleMenu.java

```diff
@@ -22,11 +24,23 @@
  */
 public class CircleMenu extends CirclePopupMenu {
 
+    /** List of all block on which a CircleMenu is active. */
+    private static ArrayList<Block> activeBlocks = new ArrayList<>();
```

Comment:
This is fixing up a corner case in circlemenu behaviour, and I think that code does not belong/has anything to do with Block.